pkgs/development/interpreters/php/default.nix

```diff
@@ -19,7 +19,8 @@ let
 
       enableParallelBuilding = true;
 
-      buildInputs = [ flex bison pkgconfig systemd ];
+      buildInputs = [ flex bison pkgconfig ]
+      ++ lib.optional stdenv.isLinux [systemd ];
 
```

Comment:
Either use `lib.optionals` (note the plural) or do not put `systemd` in a list: `lib.optional` already wraps its second argument into a list.